sdk/iotfirmwaredefense/arm-iotfirmwaredefense/CHANGELOG.md

```diff
@@ -1,15 +1,149 @@
+  - Interface CryptoCertificate no longer has parameter role
+  - Interface CryptoCertificate no longer has parameter usage
+  - Interface CryptoCertificateSummaryResource no longer has parameter expired
+  - Interface CryptoCertificateSummaryResource no longer has parameter expiringSoon
+  - Interface CryptoCertificateSummaryResource no longer has parameter pairedKeys
+  - Interface CryptoCertificateSummaryResource no longer has parameter selfSigned
+  - Interface CryptoCertificateSummaryResource no longer has parameter shortKeySize
+  - Interface CryptoCertificateSummaryResource no longer has parameter totalCertificates
+  - Interface CryptoCertificateSummaryResource no longer has parameter weakSignature
+  - Interface CryptoKey_2 no longer has parameter keySize
+  - Interface CryptoKeySummaryResource no longer has parameter pairedKeys
+  - Interface CryptoKeySummaryResource no longer has parameter privateKeys
+  - Interface CryptoKeySummaryResource no longer has parameter publicKeys
+  - Interface CryptoKeySummaryResource no longer has parameter shortKeySize
+  - Interface CryptoKeySummaryResource no longer has parameter totalKeys
+  - Interface CveResult no longer has parameter component
+  - Interface CveResult no longer has parameter cvssScore
+  - Interface CveResult no longer has parameter cvssV2Score
+  - Interface CveResult no longer has parameter cvssV3Score
+  - Interface CveResult no longer has parameter cvssVersion
+  - Interface CveResult no longer has parameter name
+  - Interface CveSummary no longer has parameter critical
+  - Interface CveSummary no longer has parameter high
+  - Interface CveSummary no longer has parameter low
+  - Interface CveSummary no longer has parameter medium
+  - Interface CveSummary no longer has parameter unknown
+  - Interface IoTFirmwareDefenseClientOptionalParams no longer has parameter $host
+  - Interface IoTFirmwareDefenseClientOptionalParams no longer has parameter endpoint
+  - Interface PairedKey no longer has parameter id
+  - Type of parameter keyType of interface CryptoKey_2 is changed from string to CryptoKeyType
+  - Type of parameter summaryType of interface CveSummary is changed from "CVE" to "CommonVulnerabilitiesAndExposures"
+  - Type of parameter info of interface ErrorAdditionalInfo is changed from Record<string, unknown> to Record<string, any>
+  - Type of parameter summaryType of interface SummaryResourceProperties is changed from "Firmware" | "CVE" | "BinaryHardening" | "CryptoCertificate" | "CryptoKey" to SummaryType
+  - Type of parameter tags of interface TrackedResource is changed from {
+        [propertyName: string]: string;
+    } to Record<string, string>
+  - Class IoTFirmwareDefenseClient no longer has parameter $host
+  - Class IoTFirmwareDefenseClient no longer has parameter apiVersion
+  - Class IoTFirmwareDefenseClient no longer has parameter subscriptionId
+  - Removed Enum KnownSummaryName
+  - Enum KnownProvisioningState no longer has value Accepted
```

Comment:
can not find any breaking change detections in spec pr https://github.com/Azure/azure-rest-api-specs/pull/33298